README.md

```diff
@@ -102,6 +104,43 @@ An opinionated, highly specific, Elixir wrapper around brod: the Erlang Kafka cl
       worker(Kaffe.Consumer, [])
       ```
 
+### Kaffe GroupMember - Batch Message Consumer
+
+  1. There are two modes for using the batch consumer. The first enables
+     an easy migration path from using the `Kaffe.Consumer`. The second
+     is more batch oriented, which allows consumers to handle a group of
+     messages for efficiency or greater context.
```

Comment:
On the line breaks, where would I see that? GitHub seems to be reformatting the text the same as for unbroken lines.